openbb_terminal/common/technical_analysis/ta_helpers.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+import pandas as pd
+
+from openbb_terminal.rich_config import console
+
+
+def check_columns(
+    data: pd.DataFrame, high: bool = True, low: bool = True, close: bool = True
+) -> Optional[str]:
+    """Returns the close columns, or None if the dataframe does not have required columns
+
+    Parameters
+    ----------
+    data: pd.DataFrame
+        The dataframe to check
+    high: bool
+        Whether to check for high column
+    low: bool
+        Whether to check for low column
+    close: bool
+        Whether to check for close column
+
+    Returns
+    ----------
+    Optional[str]
+        The name of the close column, none if df is invalid
+
+    """
+    close_col = "Adj Close" if "Adj Close" in data.columns else "Close"
+    # pylint: disable=too-many-boolean-expressions
+    if (
+        ("High" not in data.columns and high)
+        or ("Low" not in data.columns and low)
+        or (close_col not in data.columns and close)
+    ):
+        console.print(
+            "[red] Please make sure that the columns 'High', 'Low', and 'Close'"
```

Comment:
Should we also add a logging error

Reply:
Up to you. I am not too familiar with logging.

Reply:
I think we can just do something simple like `logger.error(f"Invalid columns.  data has columns {data.columns}")`. @hjoaquim @Chavithra

Reply:
I don't think that's common practice across our codebase. But if for any reason we want to see it later in the logs, we can always add it!

Reply:
My personal take is more potentiall error tracking > less.

So as long as it doesnt break